WeakAuras/GenericTrigger.lua

```diff
@@ -2946,6 +2946,9 @@ do
   function Private.CheckItemCooldowns()
     for id, _ in pairs(items) do
       local startTime, duration, enabled = GetItemCooldown(id);
+       -- In 10.26. the apis return values changed from 1,0 for enabled to true, false
+       -- We should adjust once its on all versions
+      enabled = enabled and 1 or 0
```

Comment:
this break behavior on not-retail, if enabled was 0 it's now 1

Reply:
Oh indeed.

Reply:
Fixed.